tests/Doctrine/Tests/ORM/Functional/Ticket/GH6699Test.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Tests\Models\CMS\CmsUser;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+/**
+ * @group #6699
```

Comment:
We're using the ticket id without the `#` everywhere else, we should keep it consistent here too